source/_integrations/ialarmxr.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: Antifurto365 iAlarmXR
+description: Instructions on how to integrate iAlarmsXR alarms into Home Assistant.
+ha_category:
+  - Alarm
+ha_iot_class: Local Polling
```

Comment:
```suggestion
ha_iot_class: Cloud Polling
```

As you mentioned in the integration PR, this uses the iAlarm cloud.

Reply:
Good point! I'll fix also the manifest!

Reply:
Thanks @RyuzakiKK also for docs review! 🙌 Now it's fixed, thanks